Utils.py

```diff
@@ -233,7 +233,6 @@ def set_movie_properties(json_response):
     title_header = "[B]" + str(json_response['result']['setdetails']['limits']['total']) + " " + xbmc.getLocalizedString(20342) + "[/B][CR]"
     for item in json_response['result']['setdetails']['movies']:
         art = item['art']
-        streaminfo = media_streamdetails(item['file'].encode('utf-8').lower(), item['streamdetails'])
```

Comment:
On 3/4/2015 03:37, Philipp Temminghoff wrote:
> In Utils.py
> <https://github.com/phil65/script.skin.info.service/pull/5#discussion_r25774238>:
> 
>> @@ -233,7 +233,6 @@ def set_movie_properties(json_response):
>>      title_header = "[B]" + str(json_response['result']['setdetails']['limits']['total']) + " " + xbmc.getLocalizedString(20342) + "[/B][CR]"
>>      for item in json_response['result']['setdetails']['movies']:
>>          art = item['art']
>> -        streaminfo = media_streamdetails(item['file'].encode('utf-8').lower(), item['streamdetails'])
> 
> Removing those two lines probably wasnt your intention?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phil65/script.skin.info.service/pull/5/files#r25774238>.
> 


Right.  I'm just starting to learn github, so probably did something wrong.

scott smart